internal/sourcecode/templates/nextjs/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Install dependencies only when needed
+FROM node:16-alpine AS builder
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+WORKDIR /app
+COPY . .
+RUN yarn install --frozen-lockfile
+
+# If using npm with a `package-lock.json` comment out above and use below instead
+# RUN npm ci
+
+ENV NEXT_TELEMETRY_DISABLED 1
+
+RUN yarn build
```

Comment:
This needs a little bit more work. I realize that `NUXT_PUBLIC` variables need to be set as args so I plan on adding instructions here and on the docs

Reply:
You can add build args at launch time to `fly.toml` as is [done with Rails](https://github.com/superfly/flyctl/blob/master/internal/sourcecode/scan.go#L206-L210).